examples/patterns/grid/empty-columns.html

```diff
@@ -16,16 +47,15 @@
       <div class="col-7">
         .col-7
       </div>
-      <div class="col-4 prefix-1">
-        .col-4.prefix-1
+      <div class="col-4 col-start-large-8">
+        .col-4.col-start-large-8
       </div>
   </div>
   <div class="row">
       <div class="col-7">
         .col-7
       </div>
-      <div class="col-4 suffix-1">
-        .col-4.suffix-1
+      <div class="col-4">
```

Comment:
You have removed the empty column and the contents of this grid element

Reply:
Updated as above:

![image](https://user-images.githubusercontent.com/2741678/56301552-f3041900-612f-11e9-9716-c39c9df46f70.png)